personality/voms/files/server.xml

```diff
@@ -0,0 +1,21 @@
+<Server port="8005" shutdown="SHUTDOWN">
```

Comment:
This file also doesn't seem to be referenced anywhere, but also... ncm-metaconfig.